src/quo2/foundations/colors.cljs

```diff
@@ -183,6 +183,23 @@
       (let [base-color (get-in colors-map [(keyword color) suffix])]
         (if opacity (alpha base-color (/ opacity 100)) base-color))))))
 
+(defn- opacity->hex [opacity]
+  (case opacity
+    0 "00"
+    10 "1A"
+    20 "33"
+    30 "4d"
+    40 "66"
+    50 "80"
+    60 "99"
+    70 "B3"
+    80 "CC"
+    90 "E6"
+    "FF"))
+
+(defn custom-hex-color [color light-opacity dark-opacity]
```

Comment:
Why can't we just use the `custom-color-by-theme` function? It seems to support different opacity levels based on the theme, which seems to be what you need, or not?

Reply:
`custom-color-by-theme` is not working, not sure why

Reply:
lets find out why it doesn't work, cc @Parveshdhull

Reply:
Hi @OmarBasem,
As I can see `custom-color-by-theme` is not working for you for group avatars, https://github.com/status-im/status-mobile/pull/14494/files#r1041791605
1. 50, 60 are color variants, not opacity
![image](https://user-images.githubusercontent.com/17097240/206179934-b2b6e864-d462-46ea-a3a8-113b9fce5139.png)
2. You need to provide color as `:purple` instead of hex value, then we will be able to find its appropriate variant, like :purple-50, :purple-60

Reply:
How can I do that? The value comes as hex value

Reply:
@OmarBasem, are we going to be able to get rid of this new function or do you really need it? This is only thing holding me from approving the PR, but if you think there's no way around this, I'm fine. If nothing else works, it would be nice to have a `TODO(OmarBasem)` saying something like *"custom-hex-color could be removed if XYZ is done"*.

Reply:
please use `theme-alpha` instead of creating new functions